hack/port-forward.sh

```diff
@@ -10,6 +10,9 @@ info() {
 info "MinIO on http://localhost:9000"
 kubectl -n argo port-forward pod/minio 9000:9000 &
 
+info "Prometheus on http://localhost:9090"
```

Comment:
this is incorrect - it's not Prometheus running on port 9090 - it's the metrics endpoint - so you should name it as such methinks